lib/miq_automation_engine/service_models/miq_ae_service_configuration_script.rb

```diff
@@ -1,4 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceConfigurationScript < MiqAeServiceModelBase
+    expose :run
```

Comment:
@syncrou 
Do we also need to expose the manager method?
Since that would be needed to poll the job status returned from the run method